integration

```diff
@@ -656,6 +656,7 @@
   "ryanmac8/Home-Assistant-Marta",
   "ryannazaretian/hacs-nexia-climate-integration",
   "ryanwinter/hass-rainforest-emu-2",
+  "rytilahti/homeassistant-upnp-availability",
```

Comment:
https://github.com/hacs/default/actions/runs/3724471340/jobs/6316646938#step:3:18 CI is failing

Reply:
Indeed, the `ignore: brands` for the hacs action made the checks only pass on the repository itself, so this is pending for the merging of https://github.com/home-assistant/brands/pull/3907

Reply:
https://github.com/home-assistant/brands/pull/3907 is now merged so the CI should pass *fingers crossed*.